app/views/families/_family.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="table">
+    <div class="tr">
+      <div class="td" style="justify-content: center;">  <%= family.last_name %> </div>
```

Comment:
Why is the style in the view? can we move it to a stylesheet?

Reply:
Also double space

Reply:
👌